src/internet_identity/src/stats/event_stats/tests.rs

```diff
@@ -397,6 +397,90 @@ fn should_prune_monthly_events_after_30d() {
     );
 }
 
+#[test]
+fn should_prune_at_most_100_events_24h() {
```

Comment:
As mentioned, maybe we shouldn't call "pruning" to the process of updating the aggregations?